contrib/dind/kubespray-dind.yaml

```diff
@@ -10,7 +10,7 @@ bootstrap_os: debian
 
 docker_version: latest
 
-docker_storage_options: -s overlay2 --storage-opt overlay2.override_kernel_check=true -g /dind/docker
+docker_storage_driver: overlay2
 
```

Comment:
Maybe `docker_data_root` should be set here with "/dind/docker" as value ?

Reply:
First of all, I have not much idea about this dind yaml...

BTW, AFAIK running docker-in-docker in a modern way should be: mount the host's `/var/run/docker.sock` into the container, then install the docker CLI inside the container, too. This will let the container's docker CLI using the host's `/var/run/docker.sock` and so start another new container in the same level under host.

If this is correct, the truth docker data-root should be the host's one, and so we shouldn't specify it here?

Reply:
I'm not discussing the way dind is used in this contrib, just the fact that it will break the current behaviour if `docker_data_root` is not defined here.